apps/docs/src/routes/core/ambient-light.md

```diff
@@ -2,7 +2,7 @@
 title: AmbientLight
 ---
 
-!!!module_summary title=AmbientLight|name=AmbientLight|sourcePath=lights/AmbientLight.svelte|from=core|type=component
+!!!module_summary title=AmbientLight|name=AmbientLight|sourcePath=lights/AmbientLight.svelte|from=core|type=component|relatedDocs={[{name:"three.js AmbientLight reference",url:"https://threejs.org/docs/#api/en/lights/AmbientLight"}]}
```

Comment:
Not sure how I feel about the syntax - it's a bit clunky, but I feel like it's mostly due to it all being compressed on one line for this special shorthand component syntax. Maybe it'd be easier to read if we just use normal component syntax for the module summary now that there's so many props?

Reply:
I think both ways work fine-ish. There's unfortunately no type checking in .md files so, the markdown extension way is fine for me.